components/StyledButtonSet.js

```diff
@@ -13,8 +14,11 @@ const StyledButtonItem = styled(StyledButton)`
   border-radius: 0;
   line-height: 1.5;
   flex-grow: 1;
+  border-color: ${themeGet('colors.black.200')};
```

Comment:
It seems that in the design the color used is the same than for inputs (`black.300`).

![image](https://user-images.githubusercontent.com/1556356/70062779-f973d200-15e6-11ea-9fe3-fc9304317230.png)

```suggestion
```